.github/actions/sign/action.yml

```diff
@@ -0,0 +1,121 @@
+name: GDExtension Sign
```

Comment:
It might be more appropriate in this case to just limit the script's license to this specific file and omit `THIRDPARTY.txt`. That way downstream users of this template shouldn't feel the need to redistribute the license/copyright, since this GitHub Action likely won't be distributed along with the extension binaries anyway.

Considering you've made changes to the script, your copyright notice should probably be in there as well.

```suggestion
# This file incorporates work covered by the following copyright and permission notice:  
# 
#     Copyright (c) Mikael Hermansson and Godot Jolt contributors.
#     Copyright (c) Dragos Daian.
# 
#     Permission is hereby granted, free of charge, to any person obtaining a copy of
#     this software and associated documentation files (the "Software"), to deal in
#     the Software without restriction, including without limitation the rights to
#     use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
#     the Software, and to permit persons to whom the Software is furnished to do so,
#     subject to the following conditions:
# 
#     The above copyright notice and this permission notice shall be included in all
#     copies or substantial portions of the Software.
# 
#     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
#     IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
#     FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
#     COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
#     IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
#     CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

name: GDExtension Sign
```

Reply:
Great idea, will do as you asked!